lib/views/home/widgets/tiles/tiles_widget.dart

```diff
@@ -30,8 +30,7 @@ class TilesWidget extends ConsumerWidget {
     );
   }
 
-  Padding _buildTiles(
-    BuildContext context,
+  Padding buildTiles(
```

Comment:
This too... Why are they all public now?

Reply:
Sorry that was my fault